dartagnan/src/main/java/com/dat3m/dartagnan/verification/solving/DataRaceSolver.java

```diff
@@ -49,18 +49,13 @@ private void run() throws InterruptedException, SolverException, InvalidConfigur
 		performStaticProgramAnalyses(task, analysisContext, config);
 		performStaticWmmAnalyses(task, analysisContext, config);
 
-		context = EncodingContext.of(task, analysisContext, ctx);
+		context = EncodingContext.of(task, analysisContext, ctx.getFormulaManager());
 		ProgramEncoder programEncoder = ProgramEncoder.withContext(context);
 		PropertyEncoder propertyEncoder = PropertyEncoder.withContext(context);
 		WmmEncoder wmmEncoder = WmmEncoder.withContext(context);
 		SymmetryEncoder symmetryEncoder = SymmetryEncoder.withContext(context, memoryModel, analysisContext);
```

Comment:
This is never used. Do we want to add it to the solver?

Reply:
I don't see why we shouldn't add it (apart from the general soundness issues of symmetry breaking)

Reply:
I accidentally added this in a previous commit 3804ded225682ff1682fd826e715a3f08db9a760.  I should remove this for now.  Once the issues have been solved, the symmetry encoder can be properly added.

Reply:
Symmetry encoding only fails for CAAT. It is fine here

Reply:
Then I'll add its encoding.